src/merge.rs

```diff
@@ -50,22 +50,12 @@ pub struct StructureCounts {
     /// Total number of nodes in the merged tree, excluding the
     /// root.
     pub merged_nodes: usize,
-    /// Total number of deletions to apply, local and remote.
-    pub merged_deletions: usize,
```

Comment:
Removed because it's not accurate anymore—tombstones on one side for items that don't exist on the other won't be merged anymore. We'll handle them specially when we build the list of completion ops.